apps/example/e2e/src/usecases/OffRamps.js

```diff
@@ -16,13 +16,13 @@ export default offRamps = () => {
   describe('When on Withdraw & Spend', () => {
     it('Then should have support link', async () => {
       await element(by.id('FiatExchange/scrollView')).scrollTo('bottom')
-      await expect(element(by.id('otherFundingOptions'))).toBeVisible()
+      await waitForElementById('otherFundingOptions')
     })
 
     it('Then should display total balance', async () => {
       await waitForElementById('ViewBalances')
-      await element(by.id('ViewBalances')).tap()
-      await expect(element(by.id('AssetsTokenBalance'))).toBeVisible()
+      await waitForElementById('ViewBalances', { tap: true })
```

Comment:
can this be combined with the previous line, which is the same without the `tap`?

Reply:
Changed in 2378f3bc51ed17bc5324e41edb494c2608e4a4c1!